runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPlugin.java

```diff
@@ -320,27 +469,155 @@ public void onGameStateChanged(GameStateChanged event)
 		checkStateChanged(false);
 	}
 
+	void resetPingMenu()
+	{
+		advancedPingMenuOpen = false;
+		mouseStartPosition = null;
+		pendingPartyPing = null;
+	}
+
+	private boolean pingIsSameSourceAndTarget(PartyPingData data, PartyPingData other)
```

Comment:
This strikes me as a method which should be an instance method of `PartyPingData` and not a private method in this class.